### DISCUSSION TITLE
Lost blog's css and js after deploying to Github. Help!

### DISCUSSION THREAD

**Discussion by TechLangLDY** [hexojs/hexo#5212] [Question / Answer] [answered]
I lost my Hexo blog's CSS and js after deploying to GitHub. 
I mainly have 2 problems:
+ lost css 
+ can't go to other pages. I can just visit my home page. When I tried to click "about me"/"categories"/"tags" or any other button on the home page, the website just had no change.

The problem has annoyed me for two weeks. I vaguely know the problem should be caused by my setting on the URL in the _config.yml file. I've nearly tried all answers I could find on the internet. They just didn't work. Could anyone help me? Thanks a lot!

**Comment by stevenjoezhang** [comment]
你能贴下你的源代码吗？比如配置文件什么的。你的网站试图加载 `https://techlangldy.github.io/TechLangLDY.github.io/css/index.css`，这个css 的链接显然是错误的。

**Reply by TechLangLDY** [reply, marked as answer]
问题刚刚突然解决了，我认为成功的原因应该在站点配置文件里的 `url` `root` `deploy`三个项，我把我现在的配置写出来，供有类似问题的人参考，或许可以帮到你：

```
# URL
## Set your site url here. For example, if you use GitHub Page, set url as 'https://username.github.io/project'
url: https://填你的github名.github.io
root: /
```
```
# Deployment
## Docs: https://hexo.io/docs/one-command-deployment
deploy:
  type: git
  repo: https://github.com/Github名/仓库名.git 
  branch: master
```
注意：
+ 我这是2023年弄的，其中`root`配置文件本来是没有的，我是搜网上很多对这个问题的帖子都说要设置这个才加的，或许并不用。
+ 仓库设置那里我一开始用的ssh地址，结果一直出现`Error: Spawn failed`无法推送到仓库，所以我改了https，但是我后来观察又觉得可能跟用哪个没关系，可能是我的网的问题，可能是待李断了导致的，注意要看到deploy done: git 才是真正部署成功了。
+ 我只是个小白，不一定对哦
